packages/cli/tests/commonlib/cliHelper.ts

```diff
@@ -475,4 +475,32 @@ export class CliHelper {
       }
     }
   }
+
+  static async installCLI(version: string, global: boolean, cwd: string) {
+    console.log(`install CLI with version ${version}`);
+    try {
+      if (global) {
+        const result = await execAsync(`npm install -g @microsoft/teamsfx-cli@${version}`, {
```

Comment:
it may throw error and hope you can move it to `packages\cli\tests\utils\executor.ts`